### PR TITLE
{lib}[foss/2020b] AMPtorch v0.1 w/ Python 3.8.6

### PR DESCRIPTION
**Pull request by akesandgren**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by akesandgren** [comment]
@boegelbot Please test @ generoso

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
b-an02.hpc2n.umu.se - Linux Ubuntu 20.04, x86_64, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 3.8.5
See https://gist.github.com/d0c502a4274e99f97c4f349c30e9a2bc for a full test report.

**Comment by akesandgren** [comment]
The check failures are due to a bunch of .ebs using numba-0.52.0 (which has LLVM 10.0.1) in the 2020b toolchain while most use numba-0.53.1 with LLVM 11.0.0

I.e. not this PR's fault...

File: easybuild/easyconfigs/a/AMPtorch/AMPtorch-0.1-foss-2020b.eb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+Machine-learning Package (AMP) code that seeks to provide users with
+improved performance and flexibility as compared to the original code.
+The implementation does so by benefiting from state-of-the-art machine
+learning methods and techniques to be optimized in conjunction with
+high-throughput supercomputers."""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+source_urls = ['https://github.com/ulissigroup/amptorch/archive']
+sources = [{'filename': SOURCE_TAR_GZ, 'download_filename': 'v%(version)s.tar.gz'}]
```
Comment (verdurin) [inline easybuild/easyconfigs/a/AMPtorch/AMPtorch-0.1-foss-2020b.eb:17]: @akesandgren please add the missing `checksum`.

**Review by verdurin — CHANGES_REQUESTED** [review]

**Comment by boegel** [comment]
closing this since `foss/2020b` is deprecated, see https://docs.easybuild.io/policies/toolchains

**Pull request closed without merging**